raptorWeb/raptorbot/botware.py

```diff
@@ -0,0 +1,29 @@
+from logging import getLogger
+from threading import Thread
+
+from raptorWeb import settings
+from raptorbot.discordbot.bot import raptor_bot
+from raptorbot.discordbot.util.raptorbot_settings import TOKEN
+
+LOGGER = getLogger('raptorbot.botware')
+bot_thread = Thread(target=lambda: raptor_bot.run((TOKEN)))
```

Comment:
Ideally, the bot should be able to be stopped and re-started by calling this thread from elsewhere. Confirm this,

Reply:
It seems this is not possible to do directly on the instanced bot_thread. A Thread or Process can only have it's start() method invoked once during it's lifetime. There may be some way for me to abstract this away, and create something to spawn new instances and destroy old ones whenever "start", "stop" requests are sent.

Reply:
Turns out, it can be done. I've spent a bit of time creating a BotThreadManager object. It successfully stops existing threads it is tracking, and starts new ones without a hitch. This can be integrated into any functionality in the codebase that will require a bot restart.